tests/unit/test_bundle.yaml

```diff
@@ -5,7 +5,7 @@ machines:
     series: focal
   '1':
     series: focal
-applications:
+services:
```

Comment:
should there be another test bundle for testing the case where the applications key is used as well?

Reply:
alright.   I made the test bundle a `jinja` template that tests with both 'applications' and 'services'